pkg/cloudbuild.yaml

```diff
@@ -0,0 +1,13 @@
+steps:
+- name: 'gcr.io/cloud-builders/docker'
+  args: [ 'build', '-t', 'gcr.io/$PROJECT_ID/metadata-agent-xenial-build-container', 'build/x86_64_xenial' ]
+- name: 'gcr.io/cloud-builders/docker'
+  args: [ 'run', '-v', '/workspace:/workspace', '-w', '/workspace/src/',  'gcr.io/$PROJECT_ID/metadata-agent-xenial-build-container', '/bin/bash', '-c', 'make purge && make -j4 DISTRO=xenial deb' ]
+- name: 'ubuntu:xenial'
+  args: [ 'cp', '/workspace/src/stackdriver-metadata_${_METADATA_AGENT_VERSION}.xenial.deb', '/workspace/pkg/docker/stackdriver-metadata_${_METADATA_AGENT_VERSION}.xenial.deb' ]
```

Comment:
We can also just change the `Makefile` to add an optional `DEST_DIR` for the packages to avoid this `cp` step. Or you could override `DEB_PKG` to point directly at the desired location.

Reply:
There are definitely better ways of doing it. One is to pass `"$(notdir "$@")"` to `dpkg-gencontrol`, and then add `&& cp "$(notdir "$@")" "$@"` after the `dpkg-deb` step.

Reply:
It works as-is for now, I'd like to not hyper stress the nuances, as long as it works and provides the build provenance team with what they need.

Reply:
Let's add a TODO?

Reply:
Done.